docs/addons/cloudwatch-adot-addon.md

```diff
@@ -29,7 +29,7 @@ const blueprint = blueprints.EksBlueprint.builder()
   .build(app, 'my-stack-name');
 ```
 
-Pattern # 2 : Overriding property value for different deployment Modes. This pattern deploys an ADOT collector on the namespace specified in `namespace`, name specified in `name` with `daemonset` as the mode to CloudWatch console. Deployment mode can be overridden to any of these values - `deployment`, `daemonset`, `statefulset`, `sidecar`.
+Pattern # 2 : Overriding property value for different deployment Modes. This pattern deploys an ADOT collector on the namespace specified in `namespace`, name specified in `name` with `daemonset` as the mode to visualize metrics in CloudWatch console. Deployment mode can be overridden to any of these values - `deployment`, `daemonset`, `statefulset`, `sidecar`. You can pass required metrics including custom metrics and required pod labels of applications emitting custom metrics to visualize using `metricsNameSelectors`, `podLabels` as parameters as shown below.
```

Comment:
I just had a specreq when customer was asking about deploying it for fargate. I assume the sidecar mode is for Fargate, it would help to call it out here.

Reply:
Yes that is correct. `Sidecar` is supported mainly for Fargate.